cmd/srrs/main.go

```diff
@@ -155,29 +150,59 @@ func main() {
 			}
 		}()
 
-		http.HandleFunc("/"+stateEndpoint, webapi.MakeStateHandler(pool))
-		http.HandleFunc("/"+commandEndpoint, webapi.MakeCommandHandler(pool))
-		http.HandleFunc("/"+turtleEndpoint+"/", webapi.MakeTurtleHandler(pool))
+		mux := http.DefaultServeMux
 
+		webapi.RegisterHandlers(pool, mux)
 		if *static != "" {
-			http.Handle("/", http.FileServer(http.Dir(*static)))
+			mux.Handle("/", http.FileServer(http.Dir(*static)))
 		}
 
-		logger := logger.With(zap.String("listen_addr_tcp", *tcpAddr))
+		// unsecure server
```

Comment:
Why are you using it, when it is an unsecure server?

Reply:
we need it for development/testing, but also we probably want to fallback